documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -114,7 +114,7 @@ An early access feature gate provides an opportunity for assessment before its f
 [id='ref-operator-continue-reconciliation-on-manual-ru-failure-feature-gate-{context}']
 === ContinueReconciliationOnManualRollingUpdateFailure feature gate
```

Comment:
We need to move this section under Stable feature gates (beta)  and say something like  "Currently, there are no alpha level feature gates."

Reply:
Good catch. Fixed. Thanks.